### PR TITLE
Handle Saving employer Levy In Acceptance Test

### PR DESCRIPTION
**Pull request by samuel-dev-dba**

### CODE REVIEW

File: src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Steps/EndToEndStepsBase.cs [hunk 1]
```diff
@@ -192,6 +194,24 @@ protected async Task SaveTestCommitments()
             await DataContext.SaveChangesAsync();
         }
 
+        protected async Task SaveTestEmployerHasLevyAccount(Employer employer)
```
Comment (davehowlett) [inline src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Steps/EndToEndStepsBase.cs]: I don't understand the method name. Should it be something like SaveLevyAccount?

Comment (samuel-dev-dba) [reply src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Steps/EndToEndStepsBase.cs]: Save the current Test Employer Object as a Levy Account. I will change  the method name

Comment (samuel-dev-dba) [reply src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Steps/EndToEndStepsBase.cs]: Done

File: src/SFA.DAS.Payments.AcceptanceTests.Core/Automation/TestSession.cs [hunk 2]
```diff
@@ -14,13 +15,18 @@ public class TestSession
         public long Ukprn { get; private set; }
         public List<Learner> Learners { get; }
         public Learner Learner => Learners.FirstOrDefault();
+        public Employer Employer => Employers.FirstOrDefault();
```
Comment (carlbm) [inline src/SFA.DAS.Payments.AcceptanceTests.Core/Automation/TestSession.cs]: Does it make sense to use Single here? If you have multiple employers in the list it's probably an error in the logic to call this property

File: src/SFA.DAS.Payments.AcceptanceTests.Core/Automation/TestSession.cs [hunk 3]
```diff
@@ -14,13 +15,18 @@ public class TestSession
...
         public long JobId { get; private set; }
         public DateTime IlrSubmissionTime { get; set; }
         public bool AtLeastOneScenarioCompleted { get; private set; }
+        public List<Employer> Employers { get; }
+
         //private static ConcurrentDictionary<string, ConcurrentBag<TestSession>> Sessions { get;  } = new ConcurrentDictionary<string, ConcurrentBag<TestSession>>();  //TODO: will need to be refactored at some point
         private readonly Random random;
         private readonly Faker<Course> courseFaker;
         private static readonly ConcurrentBag<long> allLearners = new ConcurrentBag<long>();
+
```
Comment (carlbm) [inline src/SFA.DAS.Payments.AcceptanceTests.Core/Automation/TestSession.cs]: Too much whitespace :)

Comment (samuel-dev-dba) [reply src/SFA.DAS.Payments.AcceptanceTests.Core/Automation/TestSession.cs]: done

File: src/SFA.DAS.Payments.AcceptanceTests.Core/Automation/TestSession.cs (hunk 2 above)
Comment (samuel-dev-dba) [reply src/SFA.DAS.Payments.AcceptanceTests.Core/Automation/TestSession.cs]: done

**Pull request merged**